src/components/dashboard/HeaderDasboard.tsx

```
@@ -85,31 +85,46 @@ export default function HeaderDasboard() {
             <button
               onClick={() => setSidebarOpen(true)}
-              className="text-orange-700 mr-4 md:hidden"
+              className="text-amber-700 dark:text-amber-400 mr-4 md:hidden"
             >
               <Menu className="w-6 h-6" />
             </button>
+            <Link
+              href="/"
+              className="text-xl font-semibold text-amber-700 dark:text-amber-300 hover:text-amber-900"
+            ></Link>
+            <button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)